src/Sandbox.js

```diff
@@ -330,12 +367,12 @@ function packageInfoKey(env: Environment, packageInfo: PackageInfo) {
       },
     });
     if (process.env.ESY__TEST) {
-      packageInfo.__key = `${normalizedName}-${version || '0.0.0'}`;
+      packageInfo.__cachedPackageHash = `${normalizedName}-${version || '0.0.0'}`;
```

Comment:
Ah, that's made for testing purposes — if `ESY__TEST` is set then esy generates dummy keys — no hash of env/deps included. That's made so we can test output with jest snapshots on different machines. We should probably just add different branches for different platforms though and test with real hashes.